src/OpenLoco/Vehicles/Vehicle.cpp

```diff
@@ -16,16 +16,16 @@ namespace OpenLoco::Vehicles
     struct VehicleCommon : VehicleBase
     {
         uint8_t pad_24[0x24 - 0x22];
-        EntityId head;               // 0x26
-        uint32_t remainingDistance;  // 0x28
-        TrackAndDirection var_2C;    // 0x2C
-        uint16_t subPosition;        // 0x2E
-        int16_t tileX;               // 0x30
-        int16_t tileY;               // 0x32
-        uint8_t tileBaseZ;           // 0x34
-        uint8_t trackType;           // 0x35 field same in all vehicles
-        RoutingHandle routingHandle; // 0x36 field same in all vehicles
-        uint8_t var_38;              // 0x38
+        EntityId head;                       // 0x26
+        uint32_t remainingDistance;          // 0x28
+        TrackAndDirection trackAndDirection; // 0x2C
+        uint16_t subPosition;                // 0x2E
+        int16_t tileX;                       // 0x30
+        int16_t tileY;                       // 0x32
+        uint8_t tileBaseZ;                   // 0x34
```

Comment:
@duncanspumpkin this one actually WASN'T already SmallZ, should it be?

Reply:
Yes anything i wrote as tileBaseZ it was just to remind me that it is in SmallZ.

Reply:
want me to change it in this PR or leave it to you?

Reply:
Sure do it this pr

Reply:
Done